### PR TITLE
Enable tests on Windows

### PR DESCRIPTION
**Pull request by meteorcloudy**

### CODE REVIEW

**Comment by meteorcloudy** [comment]
@petroseskinder @buchgr Can you help review this change?

File: generate_workspace/src/main/java/com/google/devtools/build/workspace/output/AbstractWriter.java [hunk 1]
```diff
@@ -25,14 +25,15 @@
 public abstract class AbstractWriter {
   public abstract void write(Collection<Rule> rules);
 
+  private static final String lineSeparator = System.lineSeparator();
```
Comment (buchgr) [inline generate_workspace/src/main/java/com/google/devtools/build/workspace/output/AbstractWriter.java:28]: Is `System.lineSeparator)` equivalent to `String.format("%n")`?

**Comment by buchgr** [comment]
LGTM

File: generate_workspace/src/main/java/com/google/devtools/build/workspace/output/AbstractWriter.java (hunk 1 above)
Comment (meteorcloudy) [reply generate_workspace/src/main/java/com/google/devtools/build/workspace/output/AbstractWriter.java:28]: Yes!

**Comment by meteorcloudy** [comment]
Can we merge this if everything looks good?

**Review by petroseskinder — APPROVED** [review]

**Pull request merged**